common/rdm/RDMCommandSerializer.cpp

```diff
@@ -87,8 +48,8 @@ bool RDMCommandSerializer::PackWithParams(const RDMCommand &command,
   // The buffer pointer may not be aligned, so we incur a copy here.
   const unsigned int message_length = command.MessageLength();
   RDMCommandHeader header;
-  PopulateHeader(&header, command, message_length, source, transaction_number,
-                 port_id);
+  PopulateHeader(&header, command, message_length, command.SourceUID(),
```

Comment:
Can we have a variant of PopulateHeader that just takes a header, command and length.